src/commands/Utility/quote.ts

```diff
@@ -142,6 +149,34 @@ async function addQuote(interaction: Discord.CommandInteraction, collection: Col
 		author = interaction.options.getUser("author", true),
 		url = interaction.options.getString("url", false)
 
+	let pictureUrl = interaction.options.getString("pictureUrl", false) ?? undefined
```

Comment:
Why is null being defaulted to undefined here?

Reply:
Because of line 160, where .first() can return undefined, not null. I changed it so its null if .first() returns undefined.

Reply:
You should be checking for falsy values instead, and not add them to the db if they are so, in which case undefined or null doesn’t matter. We can’t upload undefined values to the db